### PR TITLE
10.5.0 changelog

### PR DESCRIPTION
**Pull request by mschile**
Changelog for 10.5.0.

### CODE REVIEW

**Review by AtofStryker — APPROVED** [review]

File: content/_changelogs/10.5.0.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- React 18 projects will now be scaffolded with the `cypress/react18` mount
+  registration during component setup. Addresses
+  [#23033](https://github.com/cypress-io/cypress/issues/23033).
+- The list of Vue components in the project are now available for selection when
+  generating a new spec. Addresses
+  [#22836](https://github.com/cypress-io/cypress/issues/22836).
+- If the user logs in to the Dashboard from the app, but the current project
+  does not have a project ID, the user will now be able to enter the connect
+  project workflow directly. Addresses
+  [#23022](https://github.com/cypress-io/cypress/pull/23022)
```
Comment (AtofStryker) [inline content/_changelogs/10.5.0.md:28]:
```suggestion
  [#23022](https://github.com/cypress-io/cypress/issues/23022)
```

File: content/_changelogs/10.5.0.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  [#22693](https://github.com/cypress-io/cypress/pull/22693).
+- In the Launchpad, the project's name will no longer link to the 'Choose
+  Testing Type' step of the launchpad. This behavior was removed because in many
+  situations it is not valid to go back to the testing type picker and in the
+  situation where it is valid, there's already a 'Switch testing types' link on
+  the same page. Fixes
+  [#21911](https://github.com/cypress-io/cypress/pull/21911).
+- CT projects using Vite v3 work with Node versions >=17. Fixes
+  [#23042](https://github.com/cypress-io/cypress/pull/23042) &
+  [#23114](https://github.com/cypress-io/cypress/pull/23114).
```
Comment (AtofStryker) [inline content/_changelogs/10.5.0.md:74]: I think these just redirect to issues?

File: content/_changelogs/10.5.0.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+**Miscellaneous:**
+
+- Updated UI styling for Cypress `about:blank` pages. Addresses
+  [#20978](https://github.com/cypress-io/cypress/issues/20978).
+- Stable Firefox versions 101 and 102 are now explicitly rendered as unsupported
+  in Windows due to an unpatched browser incompatibility in those versions.
+  Addresses [#23164](https://github.com/cypress-io/cypress/issues/23164).
+- Add a useful warning for users on the old React API, cypress/react, who are
```
Comment (mjhenkes) [inline content/_changelogs/10.5.0.md:83]: useful seems implied

```suggestion
- Add a warning for users on the old React API, cypress/react, who are
```

File: content/_changelogs/10.5.0.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  [#22272](https://github.com/cypress-io/cypress/issues/22272).
+- Webpack entry points are no longer preserved for CT Webpack projects. Fixes
+  [#23224](https://github.com/cypress-io/cypress/pull/23224)
+- Fixes an issue where session state was not being reset when changing between
+  specs in open mode. Fixes
+  [#23146](https://github.com/cypress-io/cypress/pull/23146)
+- Add delay to header tooltips so they don't pop open unless hovered on. Fixes
+  [#23259](https://github.com/cypress-io/cypress/pull/23259)
+- Fixed an edge-case bug where Cypress would hang in run-mode if an error was
+  through from a test:before:run event handler. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:51]:
```suggestion
  thrown from a `test:before:run event handler`. Fixes
```

File: content/_changelogs/10.5.0.md (hunk 3 above)
Comment (mschile) [reply content/_changelogs/10.5.0.md]: Updated [b24000b](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/b24000bfc44224a88960308c6ca23b16ca9f2a23)

File: content/_changelogs/10.5.0.md (hunk 2 above)
Comment (mschile) [reply content/_changelogs/10.5.0.md]: Updated [b24000b](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/b24000bfc44224a88960308c6ca23b16ca9f2a23)

**Review by mjhenkes — APPROVED** [review]

File: content/_changelogs/10.5.0.md (hunk 1 above)
Comment (mschile) [reply content/_changelogs/10.5.0.md]: Updated [b24000b](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/b24000bfc44224a88960308c6ca23b16ca9f2a23)

File: content/_changelogs/10.5.0.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  [#23039](https://github.com/cypress-io/cypress/pull/23039).
+- Fixes an issue introduced in 10.3.0 where network logs could be missing
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:53]:
```suggestion
- Fixes an issue introduced in `10.3.0` where network logs could be missing
```

File: content/_changelogs/10.5.0.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  certain information or cause certain requests to be delayed by 500ms. Fixes
+  [#23227](https://github.com/cypress-io/cypress/pull/23227).
+- In the 'Create Blank Spec' dialog, pressing enter or return when the focus is
+  on the spec path input field will create the blank spec. Fixes
+  [#21815](https://github.com/cypress-io/cypress/pull/21815).
+- Fixes `lastMountedReactDom.unmountComponentAtNode is not a function` error
+  when using React 18 mount. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:60]:
```suggestion
- Fixed React 18 unmount Component handling to remove the error`lastMountedReactDom.unmountComponentAtNode is not a function` error. Fixed
```

File: content/_changelogs/10.5.0.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+- Fixes an issue introduced in `10.3.0` where network logs could be missing
+  certain information or cause certain requests to be delayed by 500ms. Fixes
+  [#23227](https://github.com/cypress-io/cypress/pull/23227).
+- In the 'Create Blank Spec' dialog, pressing enter or return when the focus is
+  on the spec path input field will create the blank spec. Fixes
+  [#21815](https://github.com/cypress-io/cypress/issues/21815).
+- Fixes `lastMountedReactDom.unmountComponentAtNode is not a function` error
+  when using React 18 mount. Fixes
+  [#23081](https://github.com/cypress-io/cypress/issues/23081).
+- Cypress will remember the spec filter in the 'Search Specs' field during a
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:62]:
```suggestion
- Fixed a regression introduced in Cypress 10.0 where Cypress no longer persisted the spec filter in the 'Search Specs' field during a
```

File: content/_changelogs/10.5.0.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  session and when a project is reopened. The same filter will be used in the
+  main specs list as well as the inline specs list present in the Test Runner.
+  Fixes [#21837](https://github.com/cypress-io/cypress/issues/21837).
+- Fixed an issue that could lead to infinite recursion and thus a crash when
+  running tests that make use of `cy.intercept`. Fixes
+  [#22693](https://github.com/cypress-io/cypress/issues/22693).
+- In the Launchpad, the project's name will no longer link to the 'Choose
+  Testing Type' step of the launchpad. Fixes
+  [#21911](https://github.com/cypress-io/cypress/issues/21911).
+- CT projects using Vite v3 work with Node versions >=17. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:72]:
```suggestion
- CT projects using Vite `v3` work with Node versions `>=17`. Fixes
```

File: content/_changelogs/10.5.0.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+  [#21815](https://github.com/cypress-io/cypress/issues/21815).
+- Fixes `lastMountedReactDom.unmountComponentAtNode is not a function` error
+  when using React 18 mount. Fixes
+  [#23081](https://github.com/cypress-io/cypress/issues/23081).
+- Cypress will remember the spec filter in the 'Search Specs' field during a
+  session and when a project is reopened. The same filter will be used in the
+  main specs list as well as the inline specs list present in the Test Runner.
+  Fixes [#21837](https://github.com/cypress-io/cypress/issues/21837).
+- Fixed an issue that could lead to infinite recursion and thus a crash when
+  running tests that make use of `cy.intercept`. Fixes
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:67]:
```suggestion
  running tests that make use of `cy.intercept()`. Fixes
```

File: content/_changelogs/10.5.0.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+  project workflow directly. Addresses
+  [#23022](https://github.com/cypress-io/cypress/issues/23022)
+- Adds Angular component testing support, including Angular template support and
+  support for Angular standalone components. Addresses
+  [#23089](https://github.com/cypress-io/cypress/issues/23089) &
+  [#22819](https://github.com/cypress-io/cypress/issues/22819) &
+  [#23096](https://github.com/cypress-io/cypress/issues/23096) &
+  [#23000](https://github.com/cypress-io/cypress/issues/23000) &
+  [#22991](https://github.com/cypress-io/cypress/issues/22991) &
+  [#23125](https://github.com/cypress-io/cypress/issues/23125).
```
Comment (emilyrohrbough) [inline content/_changelogs/10.5.0.md:36]: Looks like `[#22819](https://github.com/cypress-io/cypress/issues/22819)` was the epic for this work.

Comment (mschile) [reply content/_changelogs/10.5.0.md:36]: Are you saying to remove that from the list?

Comment (emilyrohrbough) [reply content/_changelogs/10.5.0.md:36]: We should be able to remove the list for the single issue

Comment (mschile) [reply content/_changelogs/10.5.0.md]: Updated [43949dc](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/43949dc619c69fddd2b3ce1922478de129e95ef5)

File: content/_changelogs/10.5.0.md (hunk 6 above)
Comment (mschile) [reply content/_changelogs/10.5.0.md]: Updated [43949dc](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/43949dc619c69fddd2b3ce1922478de129e95ef5)

**Review by emilyrohrbough — APPROVED** [review]

File: content/_changelogs/10.5.0.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+  [#22936](https://github.com/cypress-io/cypress/issues/22936).
+- Added new `testIsolation` configuration option to allow users to revert to
+  `legacy` mode when `experimentalSessionAndOrigin` is set to true. Read more
+  about [test isolation](/guides/core-concepts/test-isolation) in Cypress to
+  learn more. Addresses
+  [#22279](https://github.com/cypress-io/cypress/issues/22279).
+- React 18 projects will now be scaffolded with the `cypress/react18` mount
+  registration during component setup. Addresses
+  [#23033](https://github.com/cypress-io/cypress/issues/23033).
+- The list of Vue components in the project are now available for selection when
```
Comment (astone123) [inline content/_changelogs/10.5.0.md:22]: We should note here that this is currently only an option for projects that don't have a `specPattern` configured for component testing. Supporting that is in PR https://github.com/cypress-io/cypress/pull/23298

**Review by astone123 — CHANGES_REQUESTED** [review]
See my comment above ^

File: content/_changelogs/10.5.0.md (hunk 11 above)
Comment (emilyrohrbough) [reply content/_changelogs/10.5.0.md:22]: @astone123 Can you suggest the correct entry here

Comment (astone123) [reply content/_changelogs/10.5.0.md:22]: > A list of Vue components in the project are now available for selection when generating a new spec for component testing in projects that are using the default spec pattern.

**Review by astone123 — APPROVED** [review]
Approving with the assumption that my comment will be addressed. Thanks 👍🏻

File: content/_changelogs/10.5.0.md (hunk 11 above)
Comment (mschile) [reply content/_changelogs/10.5.0.md]: updated [d374886](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/d374886df30be68093496db775cc8e7f24ca1443)
